templates/blocks/review_item.liquid

```diff
@@ -0,0 +1,23 @@
+<div class="reviews-wrapper">
+  <div class="ratings-wrapper sc-flex-col sc-justify-center sc-align-items-center">
+    <div class="reviews-rating">
+      {%- render 'shared/icons/star' %}
```

Comment:
This snippets should passed a custom class as follows:

`{% render 'shared/icons/star', style_class: 'sc-icon-star' %}`